src/main/java/com/amazon/pvar/tspoc/merlin/solver/BackwardMerlinSolver.java

```diff
@@ -111,6 +111,12 @@ public void onInTransitionAdded(
                                             functionAllocation
                                     );
                                 }
+//                                if (initialQuery.stmt().getNode() instanceof CallNode callNode &&
+//                                        initialQuery.fact() instanceof MethodCall methodCall
+//                                ) {
+//                                    if (node.fact() instanceof Register)
+//                                    queryManager.
+//                                }
```

Comment:
Did you intend to leave this as is?

Reply:
That was by accident, will remove in a revision.